src/bot/commands/random/dad-jokes.ts

```diff
@@ -0,0 +1,60 @@
+import { Command, Input, Listener } from '@api';
+import * as request from 'request';
+import { Message } from 'discord.js';
+import { Emoji } from '@bot/libraries/emoji';
+import { Url } from 'url';
+import { monitorEventLoopDelay } from 'perf_hooks';
+import { CookieJar } from 'tough-cookie';
+const entities = require("html-entities").AllHtmlEntities;
+
+export class DadJokes extends Command {
+    constructor() {
+        super({
+            name: 'dadjoke',
+            description: 'Displays a random image of a cat.',
```

Comment:
I'm guessing this is from the random cat command? Either way this is a relatively severe issue, as it directly affects UX. Please push a fix for this when you can.